editor-guidelines.md

```diff
@@ -232,7 +232,18 @@ Upload the original image to the [gallery/originals](https://github.com/programm
 Add the image to the lessons page by editing the HTML in [lessons/index.md](https://github.com/programminghistorian/jekyll/blob/gh-pages/lessons/index.md) so that the image source matches the new image’s file location. In our previous example, the HTML line would be: `<img src="../gallery/cleaning-data-with-openrefine.png">`. The new image should appear on the [Lessons page]( http://programminghistorian.org/lessons)!
 
 
-### 8) Thank Everyone and Encourage Promotion
+### 8) Incorporate your lesson into our Twitter bot
+
+In addition to the Twitter promotion outlined below, we also make use of a Twitter bot to regularly re-advertise older lessons. In order to add the new lesson to our pipeline, you need to add it as a row in [this spreadsheet](https://docs.google.com/spreadsheets/d/1o-C-3WwfcEYWipIFb112tkuM-XOI8pVVpA9_sag9Ph8/edit#gid=1625380994). Everyone on the editorial team should have the ability to make changes, but write us if you have trouble doing so. You will need to add a new row for your lesson to the end of the table with the following fields:
```

Comment:
What about changing "Everyone on the editorial team should have the ability to make changes, but write us if you have trouble doing so." to "Everyone on the editorial team should have the ability to make changes; email the google group if you have trouble."? (gives a more concrete place to contact)